### ISSUE TITLE
lexik/translation-bundle 4.0.14 + doctrine/mongodb-odm-bundle 4.2 : The mapping file TransUnit.mongodb.xml is invalid

### ISSUE THREAD

**Issue by dlenglos** [lexik/LexikTranslationBundle#373]
Hi,

During the cache:clear, I have the following exception : 
```
The mapping file /var/www/html/vendor/lexik/translation-bundle/Resources/config/doctrine/TransUnit.mongodb.xml is invalid:                              
Line 16:0: Element '{http://doctrine-project.org/schemas/odm/doctrine-mongo  
-mapping}field', attribute 'strategy': [facet 'enumeration'] The value 'AUT  
O' is not an element of the set {'addToSet', 'pushAll', 'increment', 'set',  
 'setArray', 'atomicSet', 'atomicSetArray'}.                                 
                                                                             
Line 16:0: Element '{http://doctrine-project.org/schemas/odm/doctrine-mongo  
-mapping}field', attribute 'strategy': 'AUTO' is not a valid value of the a  
tomic type '{http://doctrine-project.org/schemas/odm/doctrine-mongo-mapping  
}storage-strategy'.                                                          
                                                                             
Line 16:0: Element '{http://doctrine-project.org/schemas/odm/doctrine-mongo  
-mapping}field', attribute 'fieldName': The attribute 'fieldName' is not al  
lowed.                                                                       
                                                                             
Line 16:0: Element '{http://doctrine-project.org/schemas/odm/doctrine-mongo  
-mapping}field', attribute 'id': The attribute 'id' is not allowed. 
```  

Will this be fixed in the next version ?
Thx

**Comment by bartmcleod** [comment]
Fixed in master

**Comment by dlenglos** [comment]
@bartmcleod thx. Will this be fixed on branch 4.4 ?

**Comment by bartmcleod** [comment]
That can be done. You can make it easy by creating a PR based on the definition currently in master. If that builds against 4.4 it should not be difficult.
